code-challenges.js

```diff
@@ -4,46 +4,345 @@
+// // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
+// // const myArray = [13, 34, 5, 10, 27, 42]
+// // const myArrayFirstnum = (myArray.slice(0,1))
+// // const numEval1 = () => {
+// // }
+// // console.log(myArrayFirstnum)
+// //I wrote the above before calling it a day.
+// // const myArray = [13, 34, 5, 10, 27, 42]
+// //I'm guessing this was the given and everything below is what I wrote.
+// //I think I was trying to answer a question that wasnt even being asked so I'll start over.
+// // const myArrayFirstnum = (myArray.slice(0,1))
+// // const numEval1 = () => {
+// // }
+// // console.log(myArrayFirstnum)
+// let myArray = [13, 34, 5, 10, 27, 42]
+// //What I need it something that counts up but only as high as the last array, so even if the array gets bigger or smaller is gives the correct number of responses
+// //I see now that what I was doing above was an attempt at getting the number of inputs in the array I think I can use now that I've thought about the question
+// // const myArrayFirstnum = (myArray.slice(0,1))
+// // const numEval1 = () => {
+// // }
+// // console.log(myArrayFirstnum)
+// //this gives me the first number
+// //I will see about making it push out the last number instead
+// const myArrayFirstnum = (myArray.pop())
+// const numEval1 = () => {
+
+// }
+// // console.log(myArrayFirstnum)
+// let totalNumArray = (myArray.push(myArrayFirstnum))
+// // console.log(myArray)
+// // console.log(totalNumArray)
+// //with the above I have save the total number of in an array in totalNumArray
+// //I need to make an array with each of the numbers 0-5
+// // for(let i = 0; i <= totalNumArray; i++){
+// //     console.log("Counting 1 6 times?")
+// // }
+// // //I counted 7 times so I'll change the <= to < and see what that does
+// // for(let i = 0; i < totalNumArray; i++){
+// //             console.log("Counting 1 6 times?")
+// //     }
+// //counted 6 times
+// // need to have it return the individual array
+// // for(let i = 0; i < totalNumArray; i++){
+// //             console.log(myArray[i])
+// //     }
+// //It spits out the array now need to change the result to only spit out odd or even
+// tnum)
+
+// const myArray = [13, 34, 5, 10, 27, 42]
+// const myArrayFirstnum = (myArray.pop(0))
+// let totalNumArray = (myArray.push(myArrayFirstnum))
+
+// for(let i = 0; i < totalNumArray; i++){
+//     if(myArray[i] % 2 !== 0){
+//         console.log("odd")
+//     }else if (myArray[i] % 2 === 0){
+//         console.log("even")
+//     }else {
+//         console.log("ERROR or just 0")
+//     }
+// }
+// //desired result of odd and even given moving on.
+//result :
+// odd
+// even
+// odd
+// even
+// odd
+// even
```

Comment:
Mission accomplished with lines 208 - 310
For the range section of the for loop (the middle part); consider using the length of the array
`i < myArray.length`